nilearn/datasets/struct.py

```diff
@@ -14,7 +14,7 @@
 from ..image import get_data, new_img_like, resampling
 from ._utils import fetch_files, get_dataset_descr, get_dataset_dir
 
-_package_directory = os.path.dirname(os.path.abspath(__file__))
+_package_directory = os.path.dirname(Path.resolve(__file__))
```

Comment:
Can you check that everywhere in this PR where you use the `resolve` method, you do

```python
Path(some_path).resolve()
```

and not

```python
Path.resolve(some_path)
```


For this one:

```suggestion
_package_directory = os.path.dirname(Path.resolve(__file__))
```

Reply:
So this also had to be done manually ?
Like how would I find where Path.resolve is used and replace it?

Reply:
yeah most likely manual changes